REQUIRE

```diff
@@ -1,5 +1,5 @@
 julia 0.5
-DataArrays 0.3.4
+DataArrays 0.5
```

Comment:
And with https://github.com/JuliaStats/DataArrays.jl/pull/252 we don't need to change `isna(::DataArray)` to `isna.(::DataArray)` yet as that will now throw a depwarn, right?